code/modules/atmospherics/machinery/portable/canister.dm

```diff
@@ -85,10 +85,15 @@
 
 /obj/machinery/portable_atmospherics/canister/Initialize(mapload, datum/gas_mixture/existing_mixture)
 	. = ..()
+
 	if(existing_mixture)
 		air_contents.copy_from(existing_mixture)
 	else
 		create_gas()
+
+	var/random_quality = rand(0, 100) / 100
+	pressure_limit = initial(pressure_limit) * (1 + 0.1 * random_quality)
```

Comment:
Why not just
```suggestion
	pressure_limit *= 1 + 0.1 * random_quality
```

Reply:
No particular reason really, I was working with the initial value as the base value and this seemed a bit more clear to me in reading.